compiler/plugins/input/TOSA/InputConversion/test/apply_pdl_patterns_tosa.mlir

```diff
@@ -1,30 +1,30 @@
 // RUN: iree-opt --pass-pipeline="builtin.module(iree-preprocessing-apply-pdl-patterns{patterns-file=%p/tosa.pdl.mlir})" %s | FileCheck %s
 
-// CHECK-LABEL:   stream.executable private @mlp_external_executable
-//       CHECK:   stream.executable.export public @mlp_external_entry_point
+// CHECK-LABEL:   stream.executable private @mlp_external_f32_f32_f32_i32_i32_i32_executable
+//       CHECK:   stream.executable.export public @mlp_external_f32_f32_f32_i32_i32_i32_entry_point
 //       CHECK:   builtin.module
-//       CHECK:     func.func private @mlp_external
+//       CHECK:     func.func private @mlp_external_f32_f32_f32_i32_i32_i32
 //  CHECK-SAME:         (memref<f32>, index, memref<f32>, index, memref<f32>, index, i32, i32, i32)
 //  CHECK-SAME:         attributes {llvm.bareptr = [true]}
-//       CHECK:     func.func @mlp_external_entry_point
+//       CHECK:     func.func @mlp_external_f32_f32_f32_i32_i32_i32_entry_point
```

Comment:
We dont need to change the entry point name. That is nested within the executable. We just need to uniquify the stream.executable name.

Reply:
Done. I left the entry point name as is, but along with the executable name, I also changed the private function call name so that we dont have potential collision in the shared library if we decide to support multiple types.